lib/dc/import/calais_fetcher.rb

```diff
@@ -62,14 +62,18 @@ def retry_calais_errors
         attempts = 0
         begin
           yield
-        rescue Faraday::Error, Timeout::Error => e
+        rescue CalaisError::RateLimitExceeded => e
+          attempts += 1
+          sleep 10
+          Rails.logger.warn 'waiting 10 seconds.  Calais rate limit exceeded'
+          retry if attempts < 5
+        rescue CalaisError::ServiceUnavailable, CalaisError::ServerError => e
           Rails.logger.warn e.message
-          return nil if e.message == 'Calais continues to expand its list of supported languages, but does not yet support your submitted content.'
-          Rails.logger.warn 'waiting 10 seconds'
+          Rails.logger.warn 'Calais server had an error.  Retrying...waiting 10 seconds'
           attempts += 1
           sleep 10
           retry if attempts < 5
-        rescue RuntimeError => e
+        rescue RuntimeError => e # Do we still need this catch?
```

Comment:
@cometman So long as the branch there is covered and that the content too large error is caught through other means we can remove it def.